networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-aws.adoc

```diff
@@ -0,0 +1,36 @@
+:_content-type: ASSEMBLY
+[id="configuring-ingress-cluster-traffic-aws"]
+= Configuring ingress cluster traffic on AWS
+include::_attributes/common-attributes.adoc[]
+:context: configuring-ingress-cluster-traffic-aws
+
+toc::[]
+
+{product-title} provides methods for communicating from outside the cluster with services running in the cluster. This method uses load balancers on AWS, specifically a Network Load Balancer (NLB) or a Classic Load Balancer (CLB). Both types of load balancers can forward the client's IP address to the node, but a CLB requires proxy protocol support, which {product-title} automatically enables.
+
+You can configure these load balancers on a new or existing AWS cluster.
+
+include::modules/nw-configuring-elb-timeouts-aws-classic.adoc[leveloffset=+1]
+
+include::modules/nw-configuring-route-timeouts.adoc[leveloffset=+2]
+
+include::modules/nw-configuring-clb-timeouts.adoc[leveloffset=+2]
+
+include::modules/nw-configuring-ingress-cluster-traffic-aws-networking-load-balancer.adoc[leveloffset=+1]
+
+include::modules/nw-aws-replacing-clb-with-nlb.adoc[leveloffset=+2]
+
+include::modules/nw-aws-nlb-existing-cluster.adoc[leveloffset=+2]
+
+[IMPORTANT]
+====
+Before you can configure an Ingress Controller NLB on a new AWS cluster, you must complete the xref:../../installing/installing_aws/installing-aws-network-customizations.adoc#installation-initializing_installing-aws-network-customizations[Creating the installation configuration file] procedure.
+====
+
+include::modules/nw-aws-nlb-new-cluster.adoc[leveloffset=+2]
+
+== Additional resources
```

Comment:
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies